### PR TITLE
Add the same workspace

### PR DESCRIPTION
**Pull request by VyacheslavDudin**

### CODE REVIEW

File: backend/src/services/workspaceService.ts [hunk 1]
```diff
@@ -6,8 +7,18 @@ import { IClientCreateWorkspace } from '../common/models/workspace/IClientCreate
...
 export const createWorkspace = async (data: IClientCreateWorkspace): Promise<IWorkspaceResponse> => {
+  try {
+    const { name } = data;
+    const isWorkspaceExist = await getCustomRepository(WorkspaceRepository).findByName(name);
+    if (isWorkspaceExist) {
+      throw new Error('This workspace name is already exists');
+    }
+  } catch (err) {
+    throw new CustomError(500, 'This workspace is already exists! Please, choose the other name for your workspace.', ErrorCode.WorkspaceAlreadyExists, err);
+  }
```
Comment (AlikRakhmonov) [inline backend/src/services/workspaceService.ts:21]: bad code

File: backend/src/services/workspaceService.ts [hunk 2]
```diff
@@ -6,8 +7,18 @@ import { IClientCreateWorkspace } from '../common/models/workspace/IClientCreate
...
 import { fromCreatedWorkspaceToClient, fromClientCreateWorkspaceToCreateWorkspace } from '../common/mappers/workspace';
 import { IUser } from '../common/models/user/IUser';
+import CustomError from '../common/models/CustomError';
 
 export const createWorkspace = async (data: IClientCreateWorkspace): Promise<IWorkspaceResponse> => {
+  try {
+    const { name } = data;
+    const isWorkspaceExist = await getCustomRepository(WorkspaceRepository).findByName(name);
+    if (isWorkspaceExist) {
+      throw new Error('This workspace name is already exists');
```
Comment (AlikRakhmonov) [inline backend/src/services/workspaceService.ts:17]: why do you use Error instead of CustomError

File: backend/src/services/workspaceService.ts [hunk 3]
```diff
@@ -6,8 +7,18 @@ import { IClientCreateWorkspace } from '../common/models/workspace/IClientCreate
 import { IWorkspaceResponse } from '../common/models/workspace/IWorkspaceResponse';
 import { fromCreatedWorkspaceToClient, fromClientCreateWorkspaceToCreateWorkspace } from '../common/mappers/workspace';
 import { IUser } from '../common/models/user/IUser';
+import CustomError from '../common/models/CustomError';
 
 export const createWorkspace = async (data: IClientCreateWorkspace): Promise<IWorkspaceResponse> => {
+  try {
```
Comment (AlikRakhmonov) [inline backend/src/services/workspaceService.ts:13]: why do you need try catch?

**Comment by VyacheslavDudin** [comment]
Error is showing by toastr. It should be shown in additional error field, like while registration data is invalid in sign up form?

**Pull request merged**